src/components/cc-invoice-list/cc-invoice-list.smart.js

```diff
@@ -16,9 +20,11 @@ defineSmartComponent({
     const { apiConfig, ownerId } = context;
 
     fetchAllInvoices({ apiConfig, ownerId, signal })
-      .then((invoices) => {
-        updateComponent('state', { type: 'loaded', invoices });
-      })
+      .then(
+        /** @param {Invoice[]} invoices */
```

Comment:
question: can we add the type annotations on `fetchFullInvoice` later in the file instead?

Reply:
Yes, I plan on creating issues for stuff we need to type check (libs, etc.). I'll do it once we've merged all of this